includes/Modules/Sign_In_With_Google/Hashed_User_ID.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Class Google\Site_Kit\Modules\Sign_In_With_Google\Hashed_User_ID
+ *
+ * @package   Google\Site_Kit\Modules\Sign_In_With_Google
+ * @copyright 2024 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Modules\Sign_In_With_Google;
+
+use Google\Site_Kit\Core\Storage\User_Setting;
+
+/**
+ * Class representing the hashed Google user ID.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+final class Hashed_User_ID extends User_Setting {
```

Comment:
It looks like this isn't registered, but maybe we should for consistency? We could also implement the `sanitize_callback`.